_po/ja/how-to/synonym-expansion.po

```diff
@@ -0,0 +1,304 @@
+msgstr "### 既に存在する同義語に新たに同義語を追加する"
+
+msgid ""
+"If we want to add new synonyms in the existing one, we update the existing rec"
+"ord.\n"
+"For example, if we also want to match \"imitate\" when we use \"copy\" as a search"
+" key word."
+msgstr ""
+"既に存在する同義語に新たな同義語を追加したい場合は、既存のレコードを更新します。例えば、検索キーワードに「コピー」を使った時に、「偽物」もマッチしてほしい場"
+"合。"
+
+msgid ""
+"```sql\n"
+"UPDATE synonyms SET synonyms = array_append(synonyms, 'imitate') WHERE term = "
+"'copy';\n"
+"UPDATE synonyms SET synonyms = array_append(synonyms, 'imitate') WHERE term = "
+"'replicate';\n"
+"UPDATE synonyms SET synonyms = array_append(synonyms, 'imitate') WHERE term = "
+"'simulate';\n"
+"```"
+msgstr ""
+"```sql\n"
+"UPDATE synonyms SET synonyms = array_append(synonyms, '偽物') WHERE term = 'コピー'"
+";\n"
+"UPDATE synonyms SET synonyms = array_append(synonyms, '偽物') WHERE term = 'レプリカ"
+"';\n"
+"UPDATE synonyms SET synonyms = array_append(synonyms, '偽物') WHERE term = 'シミュレ"
+"ート';\n"
+"```"
+
+msgid ""
+"\"imitate\" is new synonym. Therefore we add record into `synonyms` by using `IN"
+"SERT` as below."
+msgstr "「偽物」は新しい同義語なので、以下のように`INSERT`を使って`synonyms`にレコードを追加します。"
+
+msgid ""
+"```sql\n"
+"INSERT INTO synonyms (term, synonyms) VALUES ('imitate', ARRAY['imitate', 'cop"
+"y', 'replicate', 'simulate']);\n"
+"```"
+msgstr ""
+"```sql\n"
+"INSERT INTO synonyms (term, synonyms) VALUES ('偽物', ARRAY['偽物', 'コピー', 'レプリカ',"
+" 'シミュレート']);\n"
+"```"
+
+msgid "#### Modify exist synonyms"
+msgstr "#### 既にある同義語の修正"
+
+msgid ""
+"If we want to modify an existing record, we can modify it with UPDATE statemen"
+"t.\n"
+"For example, if we want to modify \"Windows\" to \"Window\", we do as below."
+msgstr "既に存在するレコードを修正したい場合は、それらを更新します。例えば、「ウィンドウ」を「ウインドウ」に修正したい場合、以下のようにします。"
+
+msgid ""
+"```sql\n"
+"UPDATE synonyms SET synonyms = array_replace(synonyms, 'Windows', 'Window') WH"
+"ERE term = 'display' OR term = 'video display' OR term = 'Windows';\n"
+"UPDATE synonyms SET term = 'Window' WHERE term = 'Windows';\n"
+"```"
+msgstr ""
+"```sql\n"
+"UPDATE synonyms SET synonyms = array_replace(synonyms, 'ウィンドウ', 'ウインドウ') WHERE"
+" term = 'ディスプレイ' OR term = 'ビデオディスプレイ' OR term = 'ウィンドウ';\n"
+"UPDATE synonyms SET term = 'ウィンドウ' WHERE term = 'ウインドウ';\n"
+"```"
+
+msgid "#### Delete synonyms"
+msgstr "#### 同義語の削除"
+
+msgid ""
+"If we want to delete existing synonyms, we can delete record from `synonyms`.\n"
+"For example, if we want to delete \"Window\" from synonyms, we as below."
+msgstr "同義語を削除したい場合は、`synonyms`からレコードを削除します。例えば、synonymsから「ウィンドウ」を削除したい場合は、以下のようにします。"
+
+msgid ""
+"```sql\n"
+"UPDATE synonyms SET synonyms = array_remove(synonyms, 'Window');\n"
+"DELETE synonyms WHERE term = 'Window';\n"
+"```"
+msgstr ""
+"```sql\n"
+"UPDATE synonyms SET synonyms = array_remove(synonyms, 'ウィンドウ');\n"
+"DELETE synonyms WHERE term = 'ウィンドウ';\n"
+"```"
+
+msgid "### How to search of synonyms"
+msgstr "### 同義語検索の方法"
+
+msgid ""
+"To search of synonyms uses [`pgroonga_query_expand` function][pgroonga_query_e"
```

Comment:
**Who** uses synonym to search?
I couldn't find the answer in this sentence....

I think that _we_ use synonym to search.
In this section, doing searching tutorial is _we_.

Note that I suggested sentence:

To search of synonyms, _we use_ [`pgroonga_query_expand` function][pgroonga_query_expand]

is not correspond to Japanese sentence one-by-one.

Reply:
Ah...
Sorry, I misunderstood.

I'll fix as below.

`We can use [`pgroonga_query_expand` function][pgroonga_query_expand] to search of synonyms.`